### ISSUE TITLE
Activejobs' table needs interactive semantics for accessibility

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#426]
While adding a new feature, I ran into this issue with the active jobs table. It seems to indicate the fact that the table does something 'on-click' (i.e., expands that row and shows more detailed information) but does not do it in an accessible way.  

[Here's the mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard) on the matter, though I'm sure there's w3 docs on the same.

The image shown is this table in the firefox accessibility tab.  The error string is `Clickable elements must be focusable and should have interactive semantics.`. 

![image](https://user-images.githubusercontent.com/4874123/76983238-736c6480-6913-11ea-8f69-aae6d3a3f448.png)

**Comment by ericfranz** [comment]
Another option is a different design that is easier to work with on mobile. For example, right now our progressive disclosure solution is working with datatables, where you click a parent row and child rows appear below. Instead, we could have a detail view that quickly displays over-top (think the speed of a single page app). You would lose the ability to expand (and thus possibly view) several jobs at the same time, but it might be easier to manage the separate views in an accessible way.

**Comment by ericfranz** [comment]
https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html
https://www.w3.org/TR/wai-aria-practices-1.1/#accordion

**Comment by ericfranz** [comment]
Used a button to expand details view: https://github.com/OSC/ondemand/pull/732
